# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1450192520 18000
#      Tue Dec 15 10:15:20 2015 -0500
# Branch stable
# Node ID 9fe6db1dcc4619b88e8df36518f88adb1108350f
# Parent  ef9b2a88fbb5283558c16107c93725fae0cc3184
Backout rev af5591ef9790.

* scripts/plot/util/__gnuplot_drawnow__.m: When printing, the paper properties
are modified to size all objects to 1 pixel/point. This is needed to support
the design of GL2PS. This is also done for Gnuplot to avoid toolkit specific
code in print.m and __print_parse_opts__.m. Toolkit specific code shoudl be
limited to toolkit specific m-files, i.e.__opengl_print__.m, and
__gnuplot_print__.m. The changeset being backedout is associated with bug
report #46122.

diff --git a/scripts/plot/util/__gnuplot_drawnow__.m b/scripts/plot/util/__gnuplot_drawnow__.m
--- a/scripts/plot/util/__gnuplot_drawnow__.m
+++ b/scripts/plot/util/__gnuplot_drawnow__.m
@@ -180,18 +180,18 @@ function enhanced = gnuplot_set_term (pl
           set (h, "units", units);
         end_unwind_protect
         gnuplot_pos = position_in_pixels(1:2);
         gnuplot_size = position_in_pixels(3:4);
         if (! (output_to_screen (term)
                || any (strcmp (term, {"canvas", "emf", "gif", "jpeg", ...
                                       "pbm", "png", "pngcairo", "svg"}))))
           ## Convert to inches
-          gnuplot_pos = gnuplot_pos / get (0, "screenpixelsperinch");
-          gnuplot_size = gnuplot_size / get (0, "screenpixelsperinch");
+          gnuplot_pos = gnuplot_pos / 72;
+          gnuplot_size = gnuplot_size / 72;
         endif
         if (all (gnuplot_size > 0))
           terminals_with_size = {"canvas", "emf", "epslatex", "fig", ...
                                  "gif", "jpeg", "latex", "pbm", "pdf", ...
                                  "pdfcairo", "postscript", "png", ...
                                  "pngcairo", "pstex", "pslatex", "svg", "tikz"};
           if (__gnuplot_has_feature__ ("windows_figure_position"))
             terminals_with_size{end+1} = "windows";
